projects/js-themes-toolkit/packages/liferay-theme-tasks/theme/tasks/build/sass.js

```diff
@@ -0,0 +1,58 @@
+const chalk = require('chalk');
+const fs = require('fs');
+const path = require('path');
+const PluginError = require('plugin-error');
+const replaceExtension = require('replace-ext');
+const sass = require('sass');
+const through = require('through2');
+
+const PLUGIN_NAME = 'sass';
+
+module.exports = options => through.obj((file, enc, cb) => {
+	if (file.isNull()) {
+	  return cb(null, file);
+	}
+
+	if (file.isStream()) {
+	  return cb(new PluginError(PLUGIN_NAME, 'Streaming not supported'));
+	}
+
+	if (path.basename(file.path).indexOf('_') === 0) {
+	  return cb();
+	}
+
+	if (!file.contents.length) {
+	  file.path = replaceExtension(file.path, '.css');
+	  return cb(null, file);
+	}
+
+	try {
+		const {css} = sass.renderSync({
+			data: file.contents.toString(),
+			file: file.path,
+			includePaths: [path.dirname(file.path), file.path]
```

Comment:
I think this should actually be taken from the options config.

`includePaths: options.includePaths,`

Reply:
Yeah, at this point I was just trying to reduce the problem as much as possible, so just hardcoded this to make sure I had the proper paths.

This for sure shouldn't be merged as is ;)